src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -1283,11 +1283,29 @@ components:
         events:
           description: Events emitted by a transaction.
           type: object
+          required:
```

Comment:
Why have we decided to go with `events: { items: [ ... ] }` instead of simple `events: [ ... ]`? What's the point of this nested object?